lib/rust/ensogl/core/src/display/garbage.rs

```diff
@@ -0,0 +1,106 @@
+//! A module with garbage [`Collector`].
```

Comment:
These docs should be at least slightly better, as they are confusing right now. (Also missing "a" before "garbage").